package.json

```diff
@@ -19,9 +19,13 @@
     "Jack Domleo <jackdomleo7@gmail.com> (https://github.com/jackdomleo7)",
     "Dan Spratling <dan@danspratling.dev> (https://github.com/danspratling)",
     "Savvas Stephanides (https://github.com/SavvasStephanides)",
+<<<<<<< HEAD
```

Comment:
Please revert all changes in this file and ensure it is up to date with `master`. 😊 You don't need to add yourself twice. 😉